presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1451,6 +1453,16 @@ public SystemSessionProperties(
                         "Enable execution on native engine",
                         featuresConfig.isNativeExecutionEnabled(),
                         false),
+                booleanProperty(
+                        NATIVE_EXECUTION_VALIDATION_ENABLED,
+                        "Enable native execution plan validation",
```

Comment:
Are these properties specific to POS? If no, should we make that clear somehow?

Do we need both of these properties? Why can't we have just one flag to validate the plan and always fail if validation fails?